content/en/docs/tasks/security/authentication/mtls-migration/index.md

```diff
@@ -145,11 +217,91 @@ If you can't migrate all your services to Istio (injecting Envoy sidecar), you h
 However, when configured with `PERMISSIVE` mode, no authentication or authorization checks will be performed for plaintext traffic by default.
 We recommend you use [Istio Authorization](/docs/tasks/security/authorization/authz-http/) to configure different paths with different authorization policies.
 
-## Cleanup
+### Cleanup
 
-Remove all resources.
+To remove all temperary resources created in this task:
 
 {{< text bash >}}
 $ kubectl delete ns foo bar legacy
 Namespaces foo bar legacy deleted.
 {{< /text >}}
+
+
+## Globally enable mutual TLS for the cluster
+
+This section describes how to apply the cluster-wide DestinationRule and MeshPolicy to enforce
+mutual TLS for a cluster.
+
+{{< warning >}}
+This approach is _risky_ if you have complex TLS setups.
+If you have special TLS configurations for your services or you have
+services without Envoy sidecars, we recommend you to enable mutual TLS service by service.
+{{< /warning >}}
+
+### Configure all clients to send mutual TLS traffic
+
+Run the following command to enable all Envoy sidecars to send mutual TLS traffic to the servers.
+
+{{< text bash >}}
+$ kubectl apply -f - <<EOF
+apiVersion: "networking.istio.io/v1alpha3"
+kind: "DestinationRule"
+metadata:
+  name: "default"
+  namespace: "istio-system"
+spec:
+  host: "*.local"
+  trafficPolicy:
+    tls:
+      mode: ISTIO_MUTUAL
+EOF
+{{< /text >}}
+
+The connections between services should not be interrupted.
+
+### Lock down to mTLS for the entire cluster
+
+Run the following command to enforce all Envoy sidecars to only receive mutual TLS traffic.
+
+{{< text bash >}}
+$ kubectl apply -f - <<EOF
+apiVersion: "authentication.istio.io/v1alpha1"
+kind: "MeshPolicy"
+metadata:
+  name: "default"
+spec:
+  peers:
+  - mtls: {}
+EOF
+{{< /text >}}
+
+The connections between services should not be interrupted.
+For more details, please read the
+[Authentication policy](/docs/tasks/security/authentication/authn-policy/#globally-enabling-istio-mutual-tls)
+task.
+
+### Cleanup
+
+Clean up the configuration as follows.
+
+* To disable the global mutual TLS configuration:
+
+  {{< text bash >}}
```

Comment:
I think this is failing netlify because it needs to be indented to 4 spaces.

Reply:
Done. Thanks.